doc/configuration_support.rst

```diff
@@ -11,8 +11,4 @@
 .. automodule:: universum.configuration_support
```

Comment:
to line 8:
"See also: Please see" looks a little awkward. Could you rephrase it?